src/Units/CarrierModel.java

```diff
@@ -0,0 +1,34 @@
+package Units;
+
+import java.util.Vector;
+
+import Engine.UnitActionType;
+import Units.MoveTypes.FloatHeavy;
+import Units.MoveTypes.MoveType;
+import Units.Weapons.CarrierMissiles;
+import Units.Weapons.WeaponModel;
+
+public class CarrierModel extends UnitModel
+{
+  private static final int UNIT_COST = 30000;
+  private static final int MAX_FUEL = 99;
+  private static final int IDLE_FUEL_BURN = 1;
+  private static final int VISION_RANGE = 4;
+  private static final int MOVE_POWER = 5;
+
+  private static final MoveType moveType = new FloatHeavy();
+  private static final UnitActionType[] actions = UnitActionType.COMBAT_VEHICLE_ACTIONS;
+  private static final WeaponModel[] weapons = { new CarrierMissiles() };
+
+  public CarrierModel()
+  {
+    super("Carrier", UnitEnum.CARRIER, ChassisEnum.SHIP, UNIT_COST, MAX_FUEL, IDLE_FUEL_BURN, VISION_RANGE, MOVE_POWER, moveType, actions, weapons);
+    holdingCapacity = 2;
+    UnitEnum[] carryable = { UnitEnum.B_COPTER, UnitEnum.T_COPTER, UnitEnum.FIGHTER, UnitEnum.BOMBER, UnitEnum.STEALTH, UnitEnum.BBOMB };
```

Comment:
Hm. Probs out of scope for this, but maybe carryables should be defined in terms of chassis, rather than explicit unit types?

Reply:
The thought did cross my mind.
The only time I can think of that'd be different from what we have now is with Piperunners not being able to load into Landers, but who even cares since they can't normally interact :P 
But yes, probably out of scope.